### ISSUE TITLE
401 on getProfiles

### ISSUE THREAD

**Issue by mkln** [tomlandia/fuckr#28]
Hi
so this is a somewhat unrelated question but I've found no other way to communicate. 
I was looking at the source and trying to scrape some data writing code inspired by yours, and I manage to get to the point of receiving a list of users (50), each with his own id and main data.
after that, when I try to post a request using https://primus.grindr.com/2.0/getProfiles I get a 401 response. 
I can still request a new user list after I get the 401, and that works fine. 
Any ideas? 
I'm sure creating an issue here is not the appropriate way to ask, but I found no alternatives.
Thanks!

**Comment by tomlandia** [comment]
So I guess `/nearbyProfiles` works unauthenticated (up to 50 profiles only vs 500 authenticated, and without distances) and `/getProfiles` doesn't. See this [unofficial Grindr API documentation](https://github.com/tomlandia/fuckr/blob/master/unofficial-grindr-api-documentation.md#authentication) to implement authentication. Since Grindr uses captchas, you should probably login manually and copy the authentication token into your script.

**Comment by mkln** [comment]
Well, that's what I think I'm doing. I got the auth token from a browser with the profileId.
I use them to authenticate and I get status 200 with a Session-Id.
Then I post the request for nearbyProfiles and I get 50 of them, and I get the 401 for getProfiles

I now tried adding Session-Id in the json data I'm sending but it makes no difference.
Like so:
```
prof_post_data = {
    "Session-Id": session_id,
    "Cookies": "Session-Id=" + session_id,
    "filter": {
        "ageMinimum": "null",
        "ageMaximum": "null",
        "onlineOnly": "false", 
        "page": 1, 
        "photoOnly": "false", 
        "quantity": 500
    }, 
    "lat": **, 
    "lon": **
}
```
and this is at least partly working as the profiles I get appear to be close to the coordinates I provided.
`session_id` is the id read straight from the headers. Same results even if I remove 'Cookies' 

I'm using python-requests which gave me no problems in the past, so I'm wondering what's going on.
But I admit I'm in no way an advanced user of http stuff ;[

Thanks!
